dask/array/core.py

```diff
@@ -1489,9 +1497,13 @@ def copy(self):
         return self
 
     def to_delayed(self):
-        """ Convert Array into dask Values
+        """ Convert Array into dask Delayed objects
 
         Returns an array of values, one value per chunk.
+
+        See Also
+        --------
+            dask.array.from_delayed
```

Comment:
No need to indent (see https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections, number 9).